angular/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: guide
+title: "Frontend Development with Angular.js"
+subject: angular
+---
+
+# Frontend Development <br>with Angular.js
+
+
+AngularJS is a powerful tool to create web-applications that are dynamic and efficient. Many of the most popular websites on the web use it, including its creator, Google, and Lyft.  This framework extends the capabilities of HTML code, while making data binding easier. AngularJS will change the way you think about front-end development. 
```

Comment:
Can you hard wrap your lines to 80 characters? (i.e., insert a newline if the line length exceeds 80 characters)

There's usually a way to get your editor to do this. You might want to see this stack overflow answer for how to do it in Sublime:

http://stackoverflow.com/questions/12466430/what-is-the-usual-way-to-reformat-paragraph-in-sublime-text-2/17159320#17159320

In vim, you can use `V}gq`.